File: tier1/swift/0d482a31/Runner.swift
import Foundation

func archive(_ name: String) -> String {
    let line = "tar -cf backup.tar " + name
    let task = Process()
    task.executableURL = URL(fileURLWithPath: "/bin/sh")
    task.arguments = ["-c", line]
    try? task.run()
    return line
}

File: tier1/rust/b4db1912/runner.rs
use std::process::Command;

pub fn archive(name: &str) -> String {
    let line = format!("tar -cf backup.tar {}", name);
    let out = Command::new("sh").arg("-c").arg(line).output().unwrap();
    String::from_utf8_lossy(&out.stdout).to_string()
}
